sajibportfolio.html

```diff
@@ -1,289 +1,294 @@
-            </li>
-          </ul>
-          <button class="rsm"><a class="brt" target="_blank" href="#">Get my resume</a></button>
+  </section>
+
+  <section id="about_me">
+    <div class="top">
+      <h2 class="pta1">
+        About <br />
+        Myself
+      </h2>
+      <p class="pta2">
+        Hello I’m a data Scientist and software developer! I can help you
+        build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need
+        coded, don’t hestiate to contact me.
+      </p>
+      <div class="smd">
+        <div class="st">
+          <h2 class="stt">LET’S CONNECT</h2>
         </div>
-  
+        <ul class="fra">
+          <li class="nb1">
+            <a href="https://github.com/sajibarafatsiddiqui/" target="_blank"><img src="statics/images/github.png"
+                alt="github profilelink" /></a>
+          </li>
+          <li class="nb1">
+            <a href="https://www.linkedin.com/in/sajib-arafat-siddiqui-7aa8b366" target="_blank"><img
+                src="statics/images/Linkedin_icon.png" alt="linkedin profile link" /></a>
+          </li>
+          <li class="nb1">
+            <a href="#" target="_blank"><img src="statics/images/doits.png" alt="angel list link" /></a>
+          </li>
+          <li class="nb1">
+            <a href="#" target="_blank"><img src="statics/images/twitter.png" alt="angel list link" /></a>
+          </li>
+          <li class="nb1">
+            <a href="#" target="_blank"><img src="statics/images/M.png" alt="angel list link" /></a>
+          </li>
+        </ul>
+        <button class="rsm"><a class="brt" target="_blank" href="#">Get my resume</a></button>
       </div>
-  
-      <div id="my_skills">
-  
+
+    </div>
+
+    <div id="my_skills">
+
       <ul class="ss">
-        <li>  
-              <div id="spec_name">
-                <p class="lfs">Languages</p>
-                <img class="lfsi" src="statics/images/fleche_dessous.png">
-              </div>
-  
-              <ul id="programmes">
-                <li class="pro">
-                  <img class="img1" src="statics/images/java.png" alt="javascript image">
-                  <p class="lng1">Javascript</p>
-                </li>
-                <li class="pro">
-                  <img class="img1" src="statics/images/html.png" alt="html image">
-                  <p>HTML</p>
-                </li>
-                <li class="pro">
-                  <img class="img1" src="statics/images/css.png" alt="css image">
-                  <p>CSS</p>
-                </li>
-              </ul>
-  
-            
-         
+        <li>
+          <div id="spec_name">
+            <p class="lfs">Languages</p>
+            <img class="lfsi" src="statics/images/fleche_dessous.png">
+          </div>
+
+          <div class="programmes">
+            <ul>
```

Comment:
- [ ] As requested, Please fix the language skills section, currently the alignment of skills is not correct, your design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206856775-a70bff1e-7ff6-427b-a756-4695841b499f.png)

How it should be :point_down: 
![image](https://user-images.githubusercontent.com/97521557/206856814-0bacd89f-789c-4601-9e67-a565ce69d88b.png)

Reply:
![Uploading image.png…]()


applied. Thanks